src/main/java/com/gsheet/student/repository/GoogleSheetRepository.java

```diff
@@ -0,0 +1,34 @@
+package com.gsheet.student.repository;
+
+import com.gsheet.student.dto.google.ValueRangeUpdateDto;
+import com.gsheet.student.resource.ValueRangeResource;
+import com.gsheet.student.service.AuthTokenProvider;
+import com.gsheet.student.service.GoogleSheetsClient;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class GoogleSheetRepository {
```

Comment:
we are using repository only for classes to work with DB, as Google sheets is not DB, you should rename to Service